pip/tests-integration/test_requirements.txt

```diff
@@ -1,5 +1,5 @@
 pytest==8.2.2
-qiskit>=1.2.2,<1.3.0
+qiskit>=1.3.0,<2.0.0
```

Comment:
Now that we have some compat code in there, I wonder if there's an easy way to set up these tests to be run against 1.2 as well. But maybe that's a bigger change for another PR....

Reply:
Yeah, running tests against multiple downlevel versions of Qiskit could be interesting, but I think I'd prefer to treat that as best effort and only have CI care about latest. The recent versions seem to be changing frequently enough that I don't know we'd want to commit to keeping all the compat stuff going for that long.